content_sets_rhel7.yml

```diff
@@ -1,3 +1,6 @@
 s390x:
 - rhel-7-for-system-z-rpms
 - rhel-7-server-for-system-z-rhscl-rpms
+ppc64le:
+- rhel-7-for-ppc64le-baseos-rpms
```

Comment:
Should these be?

- rhel-7-server-for-power-le-rhscl-rpms
- rhel-7-for-power-le-rpms

Reply:
+1

Reply:
@redmark-alt  @jerboaa  Made the changes